mocktracer/mocktracer.go

```diff
@@ -48,12 +60,37 @@ func (t *MockTracer) FinishedSpans() []*MockSpan {
 	return spans
 }
 
+func (t *MockTracer) AssertStartedSpansAreFinished() error {
+	startedSpans := t.StartedSpans()
```

Comment:
instead of doing comparison, why not just remove the span from `started` when it is finished?

Reply:
That works too, will update